lib/Telemetry/src/Telemetry.h

```diff
@@ -4,28 +4,49 @@
 #include "Particle.h"
 #include "settings.h"
 
-#include "JsonMaker.h"
+#include "DataQueue.h"
 #include "Sensor.h"
 #include "SensorGps.h"
 #include "SensorThermo.h"
 
-
-
 #if OUTPUT_SERIAL_MSG
     #define DEBUG_SERIAL(x) Serial.println(x)
 #else
     #define DEBUG_SERIAL(x)
 #endif
 
-void publishMessage(String topic, String& msg) {
+DataQueue dataQ;
```

Comment:
It might be worth moving this into a `cpp` file in order to encapsulate it. Otherwise, every file that imports `Telemetry.h` will have access to this object, which may not be great.

(At the same time, I don't exactly remember whether a new copy will be created per translation unit or not. I _believe_ it does, which means we'll have multiple copies of `dataQ` if we import `Telemetry.h` in multiple files.

Reply:
Shouldn't the define guards protect against that? If not, I should make a new cpp file and possibly a whole Telemetry class to encapsulate functions common to all vehicles.

Reply:
Sort of. The define guards prevent multiple inclusions of header files in a single translation unit (ie. a single `.c` or `.cpp` file), but the compiler gets confused during the linking phase since a variable may end up defined in multiple places if you put the definition in the header.